### PR TITLE
Fix: CXL memory traffic generator bandwidth numbers

### PR DESCRIPTION
**Pull request by anandaravuri**

### CODE REVIEW

File: samples/cxl_mem_tg/cxl_tg_test.h [hunk 1]
```diff
@@ -84,7 +84,7 @@ class cxl_tg_test : public test_command {
 
   // Convert number of transactions to bandwidth (GB/s)
   double bw_calc(uint64_t xfer_bytes, uint64_t num_ticks) {
-    return (double)(xfer_bytes) /
+    return (double)(xfer_bytes)* 0.931323 /
```
Comment (tswhison) [inline samples/cxl_mem_tg/cxl_tg_test.h:87]: Use a \#define instead of the magic number.

File: samples/cxl_mem_tg/cxl_tg_test.h [hunk 2]
```diff
@@ -295,24 +308,40 @@ class cxl_tg_test : public test_command {
...
+    tg_exe_->write32(TG_SEQ_START_ADDR_WR + 0x14, 0x0);
 
     tg_exe_->write32(TG_ADDR_MODE_WR, 2);
-    tg_exe_->write32(TG_SEQ_ADDR_INCR, 0x3);
-    tg_exe_->write32(TG_SEQ_ADDR_INCR + 0x04, 0x0);
+    tg_exe_->write32(TG_ADDR_MODE_WR + 0x04 ,3 );
+    tg_exe_->write32(TG_ADDR_MODE_WR + 0x08 ,3 );
+    tg_exe_->write32(TG_ADDR_MODE_WR + 0x0C ,3 );
+    tg_exe_->write32(TG_ADDR_MODE_WR + 0x10 ,3 );
+    tg_exe_->write32(TG_ADDR_MODE_WR + 0x14 ,3 );
```
Comment (tswhison) [inline samples/cxl_mem_tg/cxl_tg_test.h:326]: Add a comment describing what's being done here.

File: samples/cxl_mem_tg/cxl_tg_test.h [hunk 3]
```diff
@@ -295,24 +308,40 @@ class cxl_tg_test : public test_command {
...
     tg_exe_->write32(TG_ADDR_MODE_RD, 2);
-
+    tg_exe_->write32(TG_ADDR_MODE_RD +0x04,3 );
+    tg_exe_->write32(TG_ADDR_MODE_RD +0x08,3 );
+    tg_exe_->write32(TG_ADDR_MODE_RD +0x0C,3 );
+    tg_exe_->write32(TG_ADDR_MODE_RD +0x10,3 );
+    tg_exe_->write32(TG_ADDR_MODE_RD +0x14,3 );
+
+    tg_exe_->write32(TG_USER_WORM_EN, 0);
+    tg_exe_->write32(TG_RETURN_TO_START_ADDR, 0);
```
Comment (tswhison) [inline samples/cxl_mem_tg/cxl_tg_test.h:344]: Add a comment describing what's being done here.

File: samples/cxl_mem_tg/cxl_tg_test.h [hunk 4]
```diff
@@ -322,10 +351,12 @@ class cxl_tg_test : public test_command {
         tg_exe_->write32(TG_BYTEEN_SEED + i * 4, 0xffffffff);
         tg_exe_->write32(TG_PPPG_SEL + i * 4, 0);
         tg_exe_->write32(TG_BYTEEN_SEL + i * 4, 0);
-        tg_exe_->write32(0x1680 + i * 4, 1);
         data_seed = ~data_seed;
     }
 
+    for (uint32_t i = 0; i < 5; i++) {
+        tg_exe_->write32(0x1680 + i * 4, 0x1a);
```
Comment (tswhison) [inline samples/cxl_mem_tg/cxl_tg_test.h:358]: Magic numbers.

File: samples/cxl_mem_tg/cxl_tg_test.h (hunk 1 above)
Comment (anandaravuri) [reply samples/cxl_mem_tg/cxl_tg_test.h]: Fixed added  #define

File: samples/cxl_mem_tg/cxl_tg_test.h (hunk 2 above)
Comment (anandaravuri) [reply samples/cxl_mem_tg/cxl_tg_test.h]: Fixed

File: samples/cxl_mem_tg/cxl_tg_test.h (hunk 3 above)
Comment (anandaravuri) [reply samples/cxl_mem_tg/cxl_tg_test.h]: Fixed

File: samples/cxl_mem_tg/cxl_tg_test.h (hunk 4 above)
Comment (anandaravuri) [reply samples/cxl_mem_tg/cxl_tg_test.h]: Fixed

**Review by tswhison — APPROVED** [review]

**Pull request merged**